app/screens/contacts-screen/contacts.tsx

```diff
@@ -116,12 +115,13 @@ export const ContactsScreen: ScreenType = ({ navigation }: Props) => {
     { skip: !hasToken },
   )
 
-  useFocusEffect(() => {
-    if (!isRefreshed) {
-      setIsRefreshed(true)
-      refetch()
-    }
-  })
+  useFocusEffect(
+    useCallback(() => {
+      startPolling(10 * 1000)
```

Comment:
As we spoke about on the call I'm not convinced on the polling.  We don't do it anywhere else in the app.  Why not just use a subscription if it's important to keep the data up to date while on the screen?